modules/greenbidsAnalyticsAdapter.js

```diff
@@ -6,7 +6,7 @@ import {deepClone, generateUUID, logError, logInfo, logWarn, getParameterByName}
 
 const analyticsType = 'endpoint';
 
-export const ANALYTICS_VERSION = '2.2.1';
+export const ANALYTICS_VERSION = '2.2.2';
```

Comment:
todo: J'aurais plutôt upgrade la mineur. C'est une nouvelle feature ?
```suggestion
export const ANALYTICS_VERSION = '2.3.0';
```

Reply:
indeed